ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -30,16 +32,52 @@
       }
     },
 
-    data: () => ({
-      loading: false,
-      interactive: {},
-      interactiveSession: {},
-      interactiveType: ''
-    }),
-
     computed: {
+      ...mapGetters('interactives', [
+        'currentInteractiveDataLoading',
+        'currentInteractive',
+        'currentInteractiveSession'
+      ]),
+
+      interactiveConfig () {
```

Comment:
I got a little confused with the name `interactiveConfig` and `localizedInteractiveConfig`. How about renaming these to `interactiveData` and `localizedInteractiveData`?